l16/dev/l16_runner.sh

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env bash
+
+RUN_DATE=$(date '+%Y-%m-%d__%H-%M-%S')
+BASE_PATH=$HOME/Workspace/lukso/network-deployment
+
+function start_catalyst {
+    echo "starting catalyst node..."
+    ./bin/geth --catalyst init $BASE_PATH/kintsugi/config/genesis.json --datadir $BASE_PATH/kintsugi/datadir/geth &>/dev/null
+    nohup ./bin/geth  \
+        --catalyst \
+        --datadir $BASE_PATH/kintsugi/datadir/geth \
+        --networkid 1337222 \
+        --http \
+        --http.api "engine,net,eth,admin,debug,miner" \
+        --http.addr 0.0.0.0 \
+		--ws \
+        --ws.api "engine,net,eth,admin,debug,miner" \
+        --ethstats atif-l15-test@dev.stats.pandora.l15.lukso.network \
```

Comment:
`atif-l15-test` hardcoded value, maybe we should use script argument to provide name? (like `$1`)